packages/orbit-components/src/Tag/index.tsx

```diff
@@ -233,48 +37,76 @@ const Tag = React.forwardRef<HTMLDivElement, Props>(
     ref,
   ) => {
     return (
-      <StyledTag
-        className="orbit-tag"
-        actionable={!!(onClick || onRemove)}
+      <div
+        className={cx(
+          "orbit-tag",
+          "font-base rounded-large p-xs box-border inline-flex items-center justify-center font-medium",
+          "duration-fast transition-[color,_background-color,_box-shadow] ease-in-out",
+          "tb:rounded-normal",
+          size === SIZES.SMALL && "text-small leading-small",
+          size === SIZES.NORMAL && "text-normal leading-normal",
+          !!(onClick || onRemove) &&
+            "cursor-pointer [&_.orbit-tag-close-container]:active:opacity-100",
+          selected && [
```

Comment:
found a visual inconsistency, tbh, not sure right now what is more correct, it's not like in the current master, but Figma missing the examples selected, probably would need to ask the designers 

![Screenshot 2024-01-25 at 11 20 07](https://github.com/kiwicom/orbit/assets/14054752/e4473344-527b-4030-9a52-668d47410a6a)
![Screenshot 2024-01-25 at 11 19 59](https://github.com/kiwicom/orbit/assets/14054752/44556236-a59d-4287-a829-dc228855f3b3)

Reply:
Yes, I noticed, but it's a bug we had before. Looking at the code, the new one is the expected. We were targeting the wrapper to apply the style but it was not applying